aimmo-game/simulation/avatar/avatar_wrapper.py

```diff
@@ -21,6 +59,11 @@ def die(self, respawn_location):
     def add_event(self, event):
         self.events.append(event)
 
+    def damage(self, amount):
+        applied_dmg = max(0, amount - self.resistance)
+        self.health -= applied_dmg
```

Comment:
It seems like it is ok https://github.com/ocadotechnology/aimmo/pull/117/files#diff-d04a139bc2877a24168490f27b3091a2L54